CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 # Changelog
+## [1.2.1] - 2021-03-24
+### Changes
+- Use inclusive language
+Use inclusive language in docs, variable names, and more
```

Comment:
This needs more explanation.

Reply:
Explain each parameter deprecated in favor of the new one.

Reply:
updated, thanks